emission/core/config.py

```diff
@@ -0,0 +1,40 @@
+import json
+import logging
+import os
+
+def get_config_data_from_env():
+    config_data_env = {
+        "url": os.getenv('DB_HOST', "localhost"),
+        "result_limit": os.getenv('DB_TS_RESULT_LIMIT', 250000)
+    }
+    return config_data_env
+
+def check_unset_env_vars():
+    config_data_env = {
+        "url": os.getenv('DB_HOST'),
```

Comment:
From our testing, it looks like these values are not formatted correctly to pull into get_database.py.